platform/mds/mds/src/main/java/org/motechproject/mds/util/ValidationUtil.java

```diff
@@ -38,6 +41,26 @@ public static void validateNoJavaKeyword(String str) {
         }
     }
 
+    /**
+     * Verifies that given string is a valid java field name. Throws {@link org.motechproject.mds.ex.entity.InvalidJavaFieldNameException}
+     * if given String is not blank and is not invalid java identifier.
```

Comment:
Something wrong in this javadoc. The method throws exception only if string is invalid java identifier, right ?